mssql_python/row.py

```
@@ -38,6 +44,55 @@ def __init__(self, cursor, description, values, column_map=None):
                 
         self._column_map = column_map
     
+    def _apply_output_converters(self, values):
+        """
+        Apply output converters to raw values.
+        
+        Args:
+            values: Raw values from the database
+            
+        Returns:
+            List of converted values
+        """
+        if not self._description:
+            return values
+        
+        converted_values = list(values)
+        
+        for i, (value, desc) in enumerate(zip(values, self._description)):
+            if desc is None or value is None:
+                continue
+            
+            # Get SQL type from description
+            sql_type = desc[1]  # type_code is at index 1 in description tuple
+            
+            # Try to get a converter for this type
+            converter = self._cursor.connection.get_output_converter(sql_type)
+            
+            # If no converter found for the SQL type but the value is a string or bytes,
+            # try the WVARCHAR converter as a fallback
+            if converter is None and isinstance(value, (str, bytes)):
+                from mssql_python.constants import ConstantsDDBC
+                converter = self._cursor.connection.get_output_converter(ConstantsDDBC.SQL_WVARCHAR.value)
+            
+            # If we found a converter, apply it
+            if converter:
+                try:
+                    # If value is already a Python type (str, int, etc.), 
+                    # we need to convert it to bytes for our converters
+                    if isinstance(value, str):
+                        # Encode as UTF-16LE for string values (SQL_WVARCHAR format)
+                        value_bytes = value.encode('utf-16-le')
+                        converted_values[i] = converter(value_bytes)
+                    else:
+                        converted_values[i] = converter(value)
+                except Exception as e:
```

All exceptions are currently suppressed silently. While it's important not to leak sensitive data, it would be beneficial to at least log the exception at a debug or warning level for troubleshooting purposes. This way, issues with converter functions can be diagnosed without exposing sensitive information.

`
except Exception:
    log('debug', 'Exception occurred in output converter', exc_info=True)
`
This will help with debugging, especially if custom converter functions are used, without leaking sensitive data.